hw/ip/sram_ctrl/data/sram_ctrl.hjson

```diff
@@ -120,6 +120,27 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "DATA.INTEGRITY",
+      desc: "Data integrity is scrambled and written to memory as part of the end-to-end integrity scheme."
```

Comment:
Since this is about the data and not the data integrity, should this be rephrased somehow? Maybe just "End-to-end data/memory integrity scheme"?

Reply:
Yeah let's do that. I'll switch the asset name to `MEM` also.

Reply:
Perfect, thanks!